validation/src/main/scala/hmda/validation/rules/lar/quality/Q068.scala

```diff
@@ -19,4 +19,5 @@ object Q068 extends EditCheck[LoanApplicationRegister] with ApplicantUtils {
         (app.coSex not equalTo(4))
     }
   }
+
```

Comment:
Same comment on these lines, seem to unnecessarily add a blank space to existing code, without changing behavior